go-controller/pkg/ovn/policy_test.go

```diff
@@ -223,7 +224,7 @@ func (p multicastPolicy) enableCmds(fExec *ovntest.FakeExec, ns string) {
 			"-- add port_group fake_uuid acls @acl",
 	})
 
-	match = getMulticastACLMatch(ns)
+	match = "ip4.src == $" + as_hash + " && ip4.mcast"
```

Comment:
nit: if it's only used once, do we need to even have a variable assigned for it on Line 206?

Reply:
Fixed, thanks.